sign-in.txt

```diff
@@ -7,3 +7,4 @@
   ________________  |  ___________________
   Brian Bansenauer  |  Bansenauer-Cascadia
   Roger L           |  Floydfix
+  Madelyn L            MadelynMLeahu
```

Comment:
Be sure to match the formatting with the vertical bar between your name and username